clang/lib/Driver/ToolChains/Linux.cpp

```diff
@@ -543,12 +543,59 @@ static void handlePAuthABI(const Driver &D, const ArgList &DriverArgs,
     CC1Args.push_back("-faarch64-jump-table-hardening");
 }
 
+static void addPointerAuthFlags(const llvm::opt::ArgList &DriverArgs,
```

Comment:
There is a lot of duplication with Darwin.cpp.  Should be moved to CommonArgs.cpp

Reply:
Thanks for suggestion, moved common code to `ToolChain::addPointerAuthFlags`, see 3afcc78d48cb49fb434478490c3ee79e7525219c